examples/digitalidentity/.env.example

```diff
@@ -0,0 +1,15 @@
+# This file is a template for defining the environment variables
+# Set the application config values here
+
+YOTI_SCENARIO_ID=xxxxxxxxxxxxxxxx
```

Comment:
**YOTI_SCENARIO_ID** is not required for Share v2.

Reply:
removed YOTI_SCENARIO_ID line

Reply:
YOTI_SCENARIO_ID removed